live-examples/wat-examples/numeric/trunc_float_to_int.wat

```diff
@@ -0,0 +1,13 @@
+(module
+  (import "console" "log" (func $log (param i32)))
+  (func $main
+
+    f32.const 10.5 ;; push an i32 onto the stack
+
+    i32.trunc_f32_s ;; convert from signed f32 to i32 (.5 will be lost)
```

Comment:
The `signed` should be on the `i32` rather than on the `f32`. And I suggest adding "rounding towards zero" to be explicit how the conversion works.

Reply:
Accepted